src/docs/self-hosted/geolocation.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Self-Hosted Geolocation'
+---
+
+Sentry can use [MaxMind's free GeoLite2 database](https://dev.maxmind.com/geoip/geoip2/geolite2/) to geolocate IP addresses, providing additional context for error events where the end-user's IP address is known, as well as for the session history of users logging into your Sentry installation. Our self-hosted offering bundles MaxMind's [`geoipupdate`](https://hub.docker.com/r/maxmindinc/geoipupdate) tool for initially downloading and periodically updating this dataset. To enable this functionality, you should [sign up for a free MaxMind account](https://www.maxmind.com/en/geolite2/signup), and then tell Sentry about your credentials via environment variables.
+
+```shell
+export GEOIPUPDATE_ACCOUNT_ID="012345"
+export GEOIPUPDATE_LICENSE_KEY="foobar"
+```
+
+By default, self-hosted instances are configured to refresh the GeoLite2 database when the `geoipupdate` service first starts, and weekly thereafter. You can change the update frequency with an additional override:
+
+```shell
+export GEOIPUPDATE_ACCOUNT_ID="012345"
+export GEOIPUPDATE_LICENSE_KEY="foobar"
+export GEOIPUPDATE_FREQUENCY="24" # hours
```

Comment:
For this to work, [this line](https://github.com/getsentry/onpremise/pull/766/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R134) should not define the value and the default value should be defined in the `.env` file.

Reply:
Ah, right. Vestige of `docker-compose.override.yml`. Updated over in the `onpremise` PR.